iroha-cpp/GrpcResponseHandler.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef GRPC_RESPONSE_HANDLER_HPP
+#define GRPC_RESPONSE_HANDLER_HPP
+
+#include "logger/logger_fwd.hpp"
+#include "logger/logger_manager_fwd.hpp"
+#include "QueryResponseHandler.hpp"
+#include "TransactionResponseHandler.hpp"
+
+
+namespace IROHA_CPP
+{
+
+
+namespace spdlog
+{
+    class logger;
+}
+
+
+class GrpcResponseHandler
+{
+    TransactionResponseHandler tx_handler_;
+    QueryResponseHandler query_handler_;
+    void handleGrpcErrors(grpc::StatusCode code);
+    logger::LoggerPtr log_;
+    std::unordered_map<int, std::string> handler_map_;
```

Comment:
I suggest not to mix: variables, method, variables - hamburgers are not healthy:D.

First methods then variables

Reply:
Outdated comment.